src/metrics/metrics_test.go

```diff
@@ -331,6 +331,7 @@ func Test_populateIndexMetricsForDatabase(t *testing.T) {
 			"schema1": collection.TableList{
 				"table1": []string{
 					"index1",
+					"index2",
```

Comment:
could you add an extra database to the mock and test if indexes are collected for both?

Reply:
Done